Sources/PrimerSDK/Classes/Extensions & Utilities/VersionUtils.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  VersionUtils.swift
+//  PrimerSDK
+//
+//  Created by Niall Quinn on 26/09/23.
+//
+
+import Foundation
+
+struct VersionUtils {
+    
+    /**
+     Returns the version string in the format `"major.minor.patch"`
+     
+     If `PrimerIntegrationOptions.reactNativeVersion` is set, it will be returned.
+     If not, the version specified as `PrimerSDKVersion` in the file `"sources/version.swift"` will be returned.
+     */
+    static var releaseVersionNumber: String? {
+        if let reactNativeVersion = Primer.shared.integrationOptions?.reactNativeVersion {
```

Comment:
Just a small question, sorry if I missed this in Slack chat, but where do we get the `reactNativeVersion` string from?

Reply:
Its provided by the React Native wrapper via `integrationOptions`, [here](https://github.com/primer-io/primer-sdk-react-native/blob/c321b86c3dc71597c2d6c798202dfc20bff326ab/ios/Sources/RNTPrimer.swift#L77).